packages/php/README.md

```diff
@@ -29,18 +29,20 @@ php artisan vendor:publish --provider="ReadMe\ServiceProvider"
 
 Note that in order to generate this URL, an API request is made to ReadMe once a day, and cached to a local file in `$COMPOSER_HOME/cache`, to retrieve your projects `base_url`. If this request to ReadMe fails, the `x-documentation-url` header will not be added to responses.
 
 If you wish to not rely on this cache, you can opt to supply a `base_log_url` option within `config/readme.php`. This value should evaluate to the public-facing URL of your ReadMe project.
 
 ### Configuration Options
+
 There are a few options you can pass in to change how the logs are sent to ReadMe. These are configured within `config/readme.php`.
 
-| Option | Use |
-| :--- | :--- |
-| development_mode | **default: false** If true, the log will be separate from normal production logs. This is great for separating staging or test data from data coming from customers |
-| blacklist | **optional** An array of keys from your API requests and responses headers and bodies that you wish to blacklist from sending to ReadMe.<br /><br />If you configure a blacklist, it will override any whitelist configuration. |
-| whitelist | **optional** An array of keys from your API requests and responses headers and bodies that you only wish to send to ReadMe. |
-| base_log_url | **optional** This is the base URL for your ReadMe project. Normally this would be `https://projectName.readme.io` or `https://docs.yourdomain.com`, however if this value is not supplied, a request to the ReadMe API will be made once a day to retrieve it. This data is cached into `$COMPOSER_HOME/cache`.
+| Option           | Use                                                                                                                                                                                                                                                                                                             |
+| :--------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| development_mode | **default: false** If true, the log will be separate from normal production logs. This is great for separating staging or test data from data coming from customers                                                                                                                                             |
+| denylist         | **optional** An array of keys from your API requests and responses headers and bodies that you wish to deny sending to ReadMe.<br /><br />If you configure a denylist, it will override any allowlist configuration.                                                                                            |
```

Comment:
```suggestion
| denylist         | **optional** An array of keys from your API requests, responses headers, and request bodies that you wish to not be sent to ReadMe.<br /><br />If you configure a denylist, it will override any allowlist configuration.                                                                                            |
```